lighthouse-core/test/lib/statistics-test.js

```diff
@@ -18,38 +17,97 @@ describe('statistics', () => {
 
       const median = 5000;
       const pODM = 3500;
-      const distribution = statistics.getLogNormalDistribution(median, pODM);
-
-      function getPct(distribution, value) {
-        return Number(distribution.computeComplementaryPercentile(value).toFixed(2));
-      }
-      assert.equal(typeof distribution.computeComplementaryPercentile, 'function');
-      assert.equal(getPct(distribution, 2000), 1.00, 'pct for 2000 does not match');
-      assert.equal(getPct(distribution, 3000), 0.98, 'pct for 3000 does not match');
-      assert.equal(getPct(distribution, 3500), 0.92, 'pct for 3500 does not match');
-      assert.equal(getPct(distribution, 4000), 0.81, 'pct for 4000 does not match');
-      assert.equal(getPct(distribution, 5000), 0.50, 'pct for 5000 does not match');
-      assert.equal(getPct(distribution, 6000), 0.24, 'pct for 6000 does not match');
-      assert.equal(getPct(distribution, 7000), 0.09, 'pct for 7000 does not match');
-      assert.equal(getPct(distribution, 8000), 0.03, 'pct for 8000 does not match');
-      assert.equal(getPct(distribution, 9000), 0.01, 'pct for 9000 does not match');
-      assert.equal(getPct(distribution, 10000), 0.00, 'pct for 10000 does not match');
+      const dist = statistics.getLogNormalDistribution(median, pODM);
+
+      expect(dist.computeComplementaryPercentile(2000)).toBeCloseTo(1.00);
```

Comment:
[`toBeCloseTo`](https://jestjs.io/docs/en/expect#tobeclosetonumber-numdigits)! 

so cool.

Reply:
gotta love that jest eh @brendankenny 😉